### PR TITLE
ui: display banner when job is pending

### PR DESCRIPTION
**Pull request by miguelgrc**
* INSPIR-3092

### CODE REVIEW

File: ui/src/jobs/components/JobStatusAlert.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col, Alert } from 'antd';
+import { JOB_STATUSES } from '../../common/constants';
+
+function JobStatusAlert({ status }) {
+  const types = { pending: 'warning', closed: 'error' };
```
Comment (harunurhan) [inline ui/src/jobs/components/JobStatusAlert.jsx]: `const ALERT_TYPES_BY_STATUS = ...` and move it above the component.

File: ui/src/jobs/components/JobStatusAlert.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  return (
+    (status === 'pending' || status === 'closed') && (
```
Comment (harunurhan) [inline ui/src/jobs/components/JobStatusAlert.jsx]: extract above as `const shouldDisplayAlert = ALERT_TYPES_BY_STATUS [status] != null`

File: ui/src/jobs/components/JobStatusAlert.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      <Row className="mb2">
+        <Col>
```
Comment (harunurhan) [inline ui/src/jobs/components/JobStatusAlert.jsx]: `Row` and `Col` is the part of detail page layout, they should stay there, you can add `mb2` to `Alert` directly.

File: ui/src/jobs/components/JobStatusAlert.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            showIcon={false}
+          />
+        </Col>
+      </Row>
+    )
+  );
+}
+
+JobStatusAlert.propTypes = {
+  status: PropTypes.oneOf(JOB_STATUSES),
```
Comment (harunurhan) [inline ui/src/jobs/components/JobStatusAlert.jsx]: well that's unnecessary to be honest, you can just `PropTypes.string`

File: ui/src/jobs/components/JobStatusAlert.jsx (hunk 3 above)
Comment (miguelgrc) [reply ui/src/jobs/components/JobStatusAlert.jsx]: Shouldn't the `mb2` stay in the detail page `Row` then?

**Pull request merged**